tools/js2c.py

```diff
@@ -353,6 +354,7 @@ def js2c(buildtype, js_modules, snapshot_tool=None, verbose=False):
         sorted_strings = sorted(magic_string_set, key=lambda x: (len(x), x))
         for idx, magic_string in enumerate(sorted_strings):
             magic_text = repr(magic_string)[1:-1]
+            magic_text = string.replace(magic_text, "\"", "\\\"")
```

Comment:
There is no need to import the string module, all strings have the `replace` method on them and also you can mix the `'` and `"` characters to remove a few backslashes.

example:
`magic_text.replace('"', '\"')`